conf/app.transportEquipment.index.routes

```diff
@@ -4,3 +4,8 @@ GET        /:arrivalId/transport-equipment/:equipmentIndex/items
 POST       /:arrivalId/transport-equipment/:equipmentIndex/items                            controllers.transportEquipment.index.GoodsReferenceController.onSubmit(arrivalId: ArrivalId,  equipmentIndex:Index, mode: Mode = NormalMode)
 GET        /:arrivalId/transport-equipment/:equipmentIndex/change-items                     controllers.transportEquipment.index.GoodsReferenceController.onPageLoad(arrivalId: ArrivalId, equipmentIndex:Index, mode: Mode = CheckMode)
 POST       /:arrivalId/transport-equipment/:equipmentIndex/change-items                     controllers.transportEquipment.index.GoodsReferenceController.onSubmit(arrivalId: ArrivalId,  equipmentIndex:Index, mode: Mode = CheckMode)
+
```

Comment:
Indentation here too please, you can look at the other routes files for examples. It just makes it nice and easy to read